File: Codewars/parityOutlier.py
def findOutlier(integers):
    ofound = 0
    efound = 0
    ocnt = 0
    ecnt = 0
    for el in integers:
        if (el % 2 == 1):
            ofound = el
            ocnt = ocnt + 1
        else:
            efound = el
            ecnt = ecnt + 1
    return(efound if ocnt > ecnt else ofound)


print("The outlier is %s" % findOutlier([1, 6, 3, 5, 7]))
print("The outlier is %s" % findOutlier([2, 6, 34, 5, 8]))

File: Codewars/parityOutlier.js
'use strict';

function findOutlier(integers) {
    let ofound;
    let efound;
    let ocnt = 0;
    let ecnt = 0;
    for (let el of integers) {
        if ( el % 2 === 1 ) {
            ofound = el;
            ocnt++;
        } else {
            efound = el;
            ecnt++;
        }
    }
    return ( (ocnt > ecnt) ? efound : ofound );
}

console.log(`The outlier is ${findOutlier([1, 6, 3, 5, 7])}`);
console.log(`The outlier is ${findOutlier([2, 6, 34, 5, 8])}`);